crates/factor-outbound-networking/src/lib.rs

```diff
@@ -145,6 +141,13 @@ impl Factor for OutboundNetworkingFactor {
     }
 }
 
+/// Returns the allowed hosts for a component.
```

Comment:
maybe just to be absolutely explicit:

```suggestion
/// Returns the allowed host strings for a component.
/// This does not resolve templated hosts.
```

or is that too much?

Reply:
Always a fan of more verbosity. Changing